crates/red_knot_python_semantic/resources/mdtest/type_of/dynamic.md

```diff
@@ -70,3 +77,26 @@ def test(x: Any, y: SomethingUnknown):
     reveal_type(y.__class__)  # revealed: type[Unknown]
     reveal_type(y.__class__.__class__.__class__.__class__)  # revealed: type[Unknown]
 ```
+
+## `type[Unknown]` has similar properties to `type[Any]`
+
+```py
+import abc
+from typing import Any
+from does_not_exist import SomethingUnknown  # error: [unresolved-import]
+
+has_unknown_type = SomethingUnknown.__class__
+reveal_type(has_unknown_type)  # revealed: type[Unknown]
+
+def test(a: type[Any], b: type[str], c: type[Any], d: type[str]):
+    """Both `type[Any]` and `type[Unknown]` are assignable to all `type[]` types"""
+    a = b
+    b = c
+    c = has_unknown_type
+    d = has_unknown_type
+
+def test2(a: type[Any], b: abc.ABCMeta):
+    """Instances of `type` subclasses are also assignable to `type[Any]` or `type[Unknown]`"""
+    b = a
+    b = has_unknown_type
```

Comment:
This test checks the opposite of what the docstring says. It checks that `type[Any]` and `type[Unknown]` are assignable to `ABCMeta`; it doesn't check the reverse.